app/components/shared/TabControl/Tab.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import styled from 'styled-components';
+
+import cssVariables from '../../../css';
+
+import { formatNumber } from '../../../lib/number';
+
+const ACTIVE_CLASS_NAME = 'active';
+
+const TabButton = styled(Link).attrs({
+  className: 'px3 pt2 pb1 border-bottom block text-decoration-none black',
+  activeClassName: ACTIVE_CLASS_NAME
+})`
+  border-bottom-width: 4px;
+  border-color: transparent;
+
+  &.${ACTIVE_CLASS_NAME} {
+    border-color: ${cssVariables['--lime']};
+  }
+`;
```

Comment:
some comments I made on this stuff while I was querying Tim’s opinion on it:

> it requires enough stuff basscss itself can’t do alone that I’ve had to use it with styled-components, but also enough stuff styled-components and basscss can’t do alone that I’ve had to implement a small custom component

(this is why I’m not just using `border-lime` in `activeClassName`)

> it’s probably mostly good but it feels inelegant
> it’s mostly complicated by react-router not having good utilities for querying or reacting to state outside of `Router.Link`